haiku-apps/haikuplot/haikuplot-1.0.0.recipe

```diff
@@ -0,0 +1,41 @@
+SUMMARY="An interface to the commandline program gnuplot"
+DESCRIPTION="HaikuPlot is an application that provides an easy-to-use \
+GUI interface to gnuplot, a popular commandline plotting tool."
+HOMEPAGE="https://github.com/HaikuArchives/HaikuPlot"
+COPYRIGHT="2015 Vale Tolpegin"
+LICENSE="MIT"
+SOURCE_URI="git+https://github.com/HaikuArchives/HaikuPlot.git#b7d0d05ad90e30ffee46c90856216b202bef4c97"
+CHECKSUM_SHA256="6a057e52af317d10bd07c4e54643a4c6f61a4b384871a6cabfb3569800879ada"
+REVISION="1"
+
+ARCHITECTURES="x86_gcc2 !x86"
+SECONDARY_ARCHITECTURES="!x86"
```

Comment:
Because I need to update to allow x86 support, just haven't gotten around to it yet